cmd/limactl/start.go

```diff
@@ -340,21 +349,26 @@ func listTemplateYAMLs() ([]TemplateYAML, error) {
 		return nil, err
 	}
 	examplesDir := filepath.Join(usrlocalsharelimaDir, "examples")
-	glob := filepath.Join(examplesDir, "*.yaml")
-	globbed, err := filepath.Glob(glob)
-	if err != nil {
-		return nil, err
-	}
+
 	var res []TemplateYAML
-	for _, f := range globbed {
-		base := filepath.Base(f)
-		if strings.HasPrefix(base, ".") {
-			continue
+	walkDirFn := func(p string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		base := filepath.Base(p)
+		if strings.HasPrefix(base, ".") || !strings.HasSuffix(base, ".yaml") {
```

Comment:
I wonder why the `strings.HasPrefix(base, ".")` check is needed.

Do you expect to have `.hidden.yaml` templates that don't show up in `limactl start --list-templates` but could still be used as `template://.hidden`?